zuul.d/jobs.yaml

```diff
@@ -2214,3 +2214,20 @@
       - OS:ubuntu-xenial
       - Arch:arm64
       - BuildType:Compile
+
+# Spark arm64 build and run unit tests periodic job
+- job:
+    name: spark-unit-test-arm64
+    parent: init-test
+    description: |
+      Apache Spark build and run unit tests on arm64 server
+    run: playbooks/spark-unit-test-arm64/run.yaml
+    nodeset: ubuntu-xenial-arm64
+    tags:
+      - Category:BigData
+      - Project:apache/spark
+      - Application:Spark@master
```

Comment:
How about introduce a tag like "hadoop version", as we test on hadoop 2.7 ..

Reply:
OK :) will add it